fmskill/plot.py

```diff
@@ -552,6 +557,30 @@ def _scatter_density(x, y, binsize: float = 0.1, method: str = "linear"):
         for i in range(len(cxy)):
             hist.append(histodata[i, j])
 
+    return hist,cxy,exy
+
+def __scatter_density(x, y, binsize: float = 0.1, method: str = "linear"):
+    """Interpolates scatter data on a 2D histogram (gridded) based on data density.
+
+    Parameters
+    ----------
+    x: np.array
+        X values e.g model values, must be same length as y
+    y: np.array
+        Y values e.g observation values, must be same length as x
+    binsize: float, optional
+        2D histogram (bin) resolution, by default = 0.1
+    method: str, optional
+        Scipy griddata interpolation method, by default 'linear'
+
+    Returns
+    ----------
+    Z_grid: np.array
+        Array with the colors based on histogram density
+    """
+
+    hist,cxy,exy=__hist2d(x, y, binsize)
```

Comment:
A single underscore is the convention to use for unused variables.
https://www.datacamp.com/tutorial/role-underscore-python#2.-ignoring-values

```suggestion
    hist,cxy,_=__hist2d(x, y, binsize)
```